rust_src/remacs-sys/lib.rs

```diff
@@ -941,13 +941,22 @@ pub struct terminal {
     pub header: Lisp_Vectorlike_Header,
 }
 
+pub enum OutputMethod {
```

Comment:
Again, I'm not comfortable with C functions being declared as directly returning Rust enums...

Reply:
@birkenfeld would you put a note in the contributing file about that?

Reply:
What's the right way to do it ?

Reply:
Named constants, I think.

Reply:
But that means that `match` does not support these names very well, right?

Reply:
`match` should work fine with constants. You just have to include a catch-all case, which is as should be.

Reply:
Right. But the compiler does not know enough to warn that you have captured all cases without the catch all. Why not manually set the value of the enum? Foo(0), Bar(1), etc.

Reply:
That is not the point; please read the text I posted at https://github.com/Wilfred/remacs/pull/447#discussion_r148715654

Reply:
Which is why I asked you to document it in our CONTRIBUTING. This is a change in expectation. Documenting why, with a link to more discussions, experiences, etc. and maybe a snippet showing what to do.

Reply:
If I can get this right, I can also do the documentation, I guess.

Reply:
Thanks!